tests/testproject/settings.py

```diff
@@ -29,3 +29,5 @@
         'APP_DIRS': True,
     },
 ]
+
+USE_TZ = False
```

Comment:
This should probably be `True` but would need more work on the tests. As that's not particularly important to this package I'm not sure it's a concern?

Reply:
It was a 3.2 change, need to add support for that as well.

Reply:
Not sure if that's really unimportant. OTOH, we don't handle dates on the server side with analytical, do we? Is any such logic all in the JavaScript code of the services, maybe?  Can we think of any tests the might make sense?